pcdet/datasets/dataset.py

```diff
@@ -124,13 +123,14 @@ def prepare_data(self, data_dict):
             assert 'gt_boxes' in data_dict, 'gt_boxes should be provided for training'
             gt_boxes_mask = np.array([n in self.class_names for n in data_dict['gt_names']], dtype=np.bool_)
 
+            calib = data_dict['calib']
             data_dict = self.data_augmentor.forward(
                 data_dict={
                     **data_dict,
                     'gt_boxes_mask': gt_boxes_mask
                 }
             )
-
+            data_dict['calib'] = calib
```

Comment:
Some datasets like Nuscene/Waymo may not have this attribute.

Reply:
I add an condition to judge whether to use this attribute.
https://github.com/yukang2017/OpenPCDet/blob/a1bd2d7bc3c093dc8a934f43b5f7a2993068d8e4/pcdet/datasets/dataset.py#L132